README.md

```diff
@@ -67,12 +67,8 @@ You can read more about this [here](https://forum.makerdao.com/t/mip116-d3m-to-s
 
 This is the oracle for sDAI which will take the input of a standard DAI price feed and convert it via the `pot.chi` factor.
 
-### SparkMigrationHelper
+## Bug Bounty
 
-Fork of the AaveMigrationHelper which has some extra features like converting from USDC to DAI automatically.
-
-## Plug and Play License
-
-This code is open source, but use within a Maker SubDAO requires a 15% revenue share of all income generated by the usage of code inside this repository to the Ethereum address labelled by the ENS domain of sparkprotocol.eth. This Plug and Play revenue share license expires on January 1st, 2026 at 00:00 UTC.
```

Comment:
Just wanna make sure, this is no longer relevant?

Reply:
Ya I don't think it matters. We haven't built very much custom code. This PnP thing needs to be rethought imo.